Lib/test/test_builtin.py

```diff
@@ -2991,7 +2991,8 @@ def test_namespace_order(self):
 
 def load_tests(loader, tests, pattern):
     from doctest import DocTestSuite
-    tests.addTest(DocTestSuite(builtins))
+    if getattr(sys, 'float_repr_style', '') == 'short':
```

Comment:
Why not just:

```suggestion
    if sys.float_repr_style == 'short':
```

In which case this attribute doesn't exist?